### ISSUE TITLE
Works on Ubuntu Mate 22.04 - with issues

### ISSUE THREAD

**Issue by wuilliam321** [jjleng/copilot-proxy#2]
Just to confirm that it works. But we really need to improve the prompt, because it usually autocomplete the whole code, include the lines that has been written before.

**Comment by wuilliam321** [comment]
One extra thing to add, this is a great model that works exactly as copilot:

https://huggingface.co/lmstudio-community/Codestral-22B-v0.1-GGUF

Just follow the instruction

user prompt:
```python
    user_prompt = (
        # codestral
        f"[SUFFIX]{suffix}[PREFIX]{prompt}"
    )

```

No system prompt:
```python
    messages = [
        {"role": "user", "content": user_prompt},
    ]
```

And nothing else!

**Comment by jjleng** [comment]
yeah, the autocomplete doesn't work great. Comparing to prompting, a better way to do autocomplete will be using the FIM template. I might take a stab at it when I have time. But feel free to contribute.
